arrow-libs/core/arrow-core/src/commonTest/kotlin/arrow/core/OptionTest.kt

```diff
@@ -143,10 +178,141 @@ class OptionTest : UnitSpec() {
       none.toList() shouldBe listOf()
     }
 
-    "firstOption" {
-      val l = listOf(1, 2, 3, 4, 5, 6)
-      l.firstOrNone() shouldBe Some(1)
-      l.firstOrNone { it > 2 } shouldBe Some(3)
+    "Iterable.firstOrNone" {
```

Comment:
Amazing work with the tests btw :slightly\_smiling\_face: Really appreciate it :pray: